R/standalone-input-names.R

```diff
@@ -0,0 +1,83 @@
+# ---
+# repo: tidymodels/workflows
+# file: standalone-input-names.R
+# last-updated: 2024-01-291
+# license: https://unlicense.org
+# requires: cli, rlang
+# ---
+
+# This file provides a portable set of helper functions for determining the
+# names of the predictor columns used as inputs into a workflow.
+
+# ## Changelog
+# 2024-01-21
+# * First version
+# 2024-01-29
```

Comment:
These logs usually have the newer entries on top. :)

In this case, though, I'd say we remove the 01-29 entry since the log is "changes" and the original hadn't made it to `main` yet.